manifests/profile/neutron/router.pp

```diff
@@ -12,14 +12,17 @@
 
   ### Router service installation
   class { '::neutron::agents::l3':
-    debug                   => $::openstack::config::debug,
-    external_network_bridge => 'brex',
-    enabled                 => true,
+    debug                    => $::openstack::config::debug,
+    external_network_bridge  => 'brex',
+    enabled                  => true,
+    agent_mode               => 'dvr_snat',
+    router_delete_namespaces => 'True',
```

Comment:
Thanks for your review.
Do you refer to the router_delete_namespace? Is it really boolean?
It is used in this context:
DEFAULT/router_delete_namespaces':         value => $router_delete_namespaces;
I thought it is a string.